src/ecall/ecall.c

```diff
@@ -2606,6 +2605,14 @@ int ecall_set_video_send_state(struct ecall *ecall, enum icall_vstate vstate)
 		return 0;
 	}
 
+    enum econn_state conn_current_state = econn_current_state(ecall->econn);
+    if (ecall->conv_type == ICALL_CONV_TYPE_ONEONONE && (ecall->update || conn_current_state == ECONN_UPDATE_RECV)) {
+        info("ecall(%p): set_video_send_state: ignored, already in update state\n", ecall);
+
+        return 0;
+    }
+
+
```

Comment:
I think this is hapening too early. We need to go through all the steps of sending propsync etc, however later, when we want to do a restart, we should skip that step...